tsconfig.json

```diff
@@ -27,6 +27,12 @@
       ],
       "@guardian/source-react-components-development-kitchen": [
         "packages/@guardian/source-react-components-development-kitchen/src/index"
+      ],
+      "@guardian/eslint-plugin-source-foundations": [
+        "packages/@guardian/eslint-plugin-source-foundations/src/index"
+      ],
+      "@guardian/eslint-plugin-source-react-components": [
+        "packages/@guardian/eslint-plugin-source-react-components/src/index"
```

Comment:
This is not strictly necessary for this change, but I thought I'd keep it in anyway as it follows the pattern of defining paths to packages in the root tsconfig. Happy to remove if it's confusing / harmful / useless 🙂

Reply:
Makes sense to follow an existing pattern, I guess?